Caltech-IPAC/firefly
# Firefly-980: support HiPS based target selection

#### [Firefly-980](https://jira.ipac.caltech.edu/browse/FIREFLY-980): support HiPS based target selection

   - Option 1: HiPS above target panel
   - Option 2: HiPS as pop up on target panel

#### Testing
- _url:_ https://fireflydev.ipac.caltech.edu/firefly-980-hips-target/firefly/firefly-dev.html
- click on `Example Js Dialog`
- Example 1
  - click on `HiPS/Target`  tab
  - click on HiPS to change target
  - change target see in change in HiPS
  - change radis, see if reflected on HiPS
- Example 2
   - click `HiPS/Target - popup`  tab
   - click on ![20x20_click](https://user-images.githubusercontent.com/6730519/166339713-d8ae03a9-7740-4c95-9356-c9d9745ad008.png) on target panel
   - do same testing but with the dialog
